src/frontend/templates/pages/game_network_groups.html

```diff
@@ -33,8 +33,8 @@ <h1 class="title">{% blocktrans with run_name=run.name %}Games for {{run_name}}
   {% for network in network_list %}
   <tr>
     <td> {{ network.name }} </td>
-    <td> <a href="{% url 'training_games_list_by_network' run.name network.name 1 %}"> {% trans "Training Games" %} </td>
-    <td> <a href="{% url 'rating_games_list_by_network' run.name network.name 1 %}"> {% trans "Rating Games" %} </td>
+    <td> <a href="{% url 'training_games_list_by_network' run.name network.name %}"> {% trans "Training Games" %} </td>
```

Comment:
what was the 1 ?

Reply:
The 1 was the page number. It is unnecessary now since we are using the GET arguments for pagination which is better and gives us the page 1 for free by default.